doc/source/rayserve/overview.rst

```diff
@@ -0,0 +1,197 @@
+RayServe: Scalable and Programmable Serving
+===========================================
+
+.. image:: logo.svg
+    :align: center
+    :height: 250px
+    :width: 400px
+
+Overview
+--------
+
+RayServe is a scalable model-serving library built on Ray Core.
+
+RayServe allows you to:
+
+- **Framework Agnostic**:Use the same toolkit to serve everything from deep learning models 
+  built with frameworks like PyTorch or TensorFlow to scikit-learn models or arbitrary business logic.
+- **Simplify MLOps**: Configure your model serving with pure Python code - no more YAMLs or 
+  JSON configs.
+-  **A/B test models** (with zero downtime) by decoupling routing logic from response handling logic.
+- Leverage built-in **batching** and **SLO awareness** to help you meet your latency objectives.
+
+Since Ray is built on Ray, RayServe also allows you to **scale to many machines**
+and allows you to leverage all of the core Ray framework so you can deploy and scale on any cloud.
+
+.. note:: 
+  If you want to try out Serve, join our `community slack <https://forms.gle/9TSdDYUgxYs8SA9e8>`_ 
+  and discuss in #serve channel.
+
```

Comment:
I'd love to see a simple example before reading why rayServe section.

Reply:
But you also have a ton of context on RayServe, so hearing why is redundant to you. I'd argue that someone just getting started will know nothing about why they should even consider RayServe and therefore it's important to bring it up. Remember, you're not necessarily the typical user. More people are just learning and therefore it's important to set that context. Besides, they can scroll a paragraph.

I did just make it more concise though ;)